modules/client-common/src/context-core.ts

```diff
@@ -107,35 +97,43 @@ export abstract class ContextCore {
   }
 
   private setNetworkDefaults() {
-    const networkName = this.network.name as SupportedNetwork;
-    if (
-      !GRAPHQL_NODES[networkName]?.length ||
-      !IPFS_NODES[networkName]?.length ||
-      !LIVE_CONTRACTS[SupportedVersion.LATEST][networkName]
-    ) {
-      throw new UnsupportedNetworkError(networkName);
+    // check network
+    const networkName = getNetworkNameByAlias(this.network.name);
+    if (!networkName) {
+      throw new UnsupportedNetworkError(this.network.name);
     }
-
+    // set graphql nodes
     if (!this.overriden.graphqlNodes) {
       this.state.graphql = ContextCore.resolveGraphql(
-        GRAPHQL_NODES[networkName],
+        getDefaultGraphqlNodes(networkName),
       );
     }
-
+    // set ipfs nodes
     if (!this.overriden.ipfsNodes) {
-      this.state.ipfs = ContextCore.resolveIpfs(IPFS_NODES[networkName]);
+      this.state.ipfs = ContextCore.resolveIpfs(
+        getDefaultIpfsNodes(networkName),
+      );
     }
-
-    for (const address of DeployedAddressesArray) {
-      if (!this.overriden[address]) {
-        let defaultAddress =
-          LIVE_CONTRACTS[SupportedVersion.LATEST][networkName][address];
+    // set contract addresses
+    for (const contractName of Object.values(ContractNames)) {
```

Comment:
`Object.values(ContractNames)` is re used, why not converting it to a global variable?